.generator/cli.py

```diff
@@ -398,22 +398,35 @@ def _create_repo_metadata_from_service_config(
         Dict: The content of the .repo-metadata.json file.
     """
     full_service_config_path = f"{source}/{api_path}/{service_config_name}"
+    with open(full_service_config_path, "r") as f:
+        service_config = yaml.safe_load(f)
+
+    publishing = service_config.get("publishing", {})
+    name_pretty = service_config.get("title", "")
+    product_documentation = publishing.get("product_documentation", "")
+    api_shortname = service_config.get("name", "").split(".")[0]
+    distribution_name = publishing.get("library_name", library_id)
```

Comment:
```suggestion
    distribution_name = library_id
```

`library_name` does not appear in the publishing section
https://github.com/googleapis/googleapis/blob/master/google/api/client.proto